packages/hardhat-plugin/src/utils/open.ts

```diff
@@ -14,5 +14,11 @@ export function open(filePath: string): void {
       command = "xdg-open";
   }
 
-  execSync(`${command} ${filePath}`);
+  try {
+    execSync(`${command} ${filePath}`, { stdio: "ignore" });
+  } catch {
+    console.log(
+      "No program found to open the file. Please open the file manually in your browser."
+    );
```

Comment:
What do you think about this alternative? As we are already printing the path, the user will know how to open it, and there's no clear expectation of auto-opening it.

```suggestion
```

Reply:
Yep, sounds good to me!